README.md

```diff
@@ -14,8 +14,9 @@ $ pip3 install -r requirements.txt
 
 Dependencies are:
 
-- `ansicolors` for stripping color codes from the output
-- `ruamel.yaml` for parsing the test's YAML metadata
+-   `ansicolors` for stripping color codes from the output
+-   `ruamel.yaml` for parsing the test's YAML metadata
+-   `tqdm` for displaying a progress bar
```

Comment:
What's up with the added indentation here?

Reply:
VSCode MD formatter did this 🤷

Reply:
alright then - don't really see why, but I don't want to fight formatters either 😄